rbc/omniscidb.py

```diff
@@ -419,62 +419,6 @@ def load_table_columnar(self, table_name, **columnar_data):
 
         thrift = self.thrift_client.thrift
         table_details = self.get_table_details(table_name)
-        use_sql = self.version[:2] < (5, 3)
```

Comment:
When a given rbc version connects to some older omniscidb version, something will break below. In order to notify that the breakage is not a bug, let's have here at least the following assert statement:
```python
        assert self.version[:2] >= (5, 6), self.version  # omniscidb is too old
```

The above note applies to all changes in this PR that may cause rbc failure when connecting to an unsupported omniscidb version.